openedx/core/djangoapps/session_inactivity_timeout/middleware.py

```diff
@@ -38,23 +42,48 @@ def process_request(self, request):
 
         # Do we have this feature enabled?
         if timeout_in_seconds:
+            frequency_time_in_seconds = getattr(settings, "SESSION_SAVE_FREQUENCY_SECONDS", 60)
             # what time is it now?
             utc_now = datetime.utcnow()
 
             # Get the last time user made a request to server, which is stored in session data
-            last_touch = request.session.get(LAST_TOUCH_KEYNAME)
+            last_touch_str = request.session.get(LAST_TOUCH_KEYNAME)
```

Comment:
This will cause a breaking change, which I am ok with. We'll inform about this. Can you follow https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html for breaking changes, adding the `!` to PR title and commit message, and update the commit message as detailed in the OEP.